code/modules/projectiles/gun.dm

```diff
@@ -239,7 +239,7 @@
 			if(!user)
 				break
 			if(!issilicon(user))
-				if( i>1 && !(src in get_both_hands(user))) //for burst firing
+				if(i>1 && !(src in get_both_hands(user))) //for burst firing
```

Comment:
```suggestion
				if(i > 1 && !(src in get_both_hands(user))) //for burst firing
```

Reply:
This PR is gonna be sticking exclusively to the spacing for conditionals, not the operators, because theres like 5000+ cases of those. Out of scope.